includes/Core/Util/BC_Functions.php

```diff
@@ -124,4 +124,76 @@ protected static function load_script_textdomain() {
 		return false;
 	}
 
+	/**
+	 * Basic implementation of the wp_sanitize_script_attributes function introduced in the WordPress version 5.7.0.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $attributes Key-value pairs representing `<script>` tag attributes.
+	 * @return string String made of sanitized `<script>` tag attributes.
+	 */
+	private static function sanitize_attributes( $attributes ) {
```

Comment:
I think this method should keep the same name as `wp_sanitize_script_attributes` as the class will attempt to call the global function by the same name first. `sanitize_attributes` doesn't exist in WP right now, but it could later, or be defined by some other userland code which could result in unexpected behavior. This should be safe as the only difference with the core method is the theme-support for html5 check. The ACs sound like maybe we shouldn't use this name and that the method would be entirely internal/private but this class doesn't support private static methods as-is.

@felixarntz Is there another reason why we would want to avoid using the core name here or is this just a misinterpretation of the ACs?

Reply:
@aaemnnosttv Yes, this should use the original WP core function name.

Reply:
Okay. Changing it to `wp_sanitize_script_attributes`. I think the original intent on the AC was to discourage use of this function externally but as @aaemnnosttv said this is not supported by the class because of callStatic.